Updating Recipe

sepandhaghighi opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

sepandhaghighi committed:
fix : update recipe

sepandhaghighi committed:
bump build number

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

sepandhaghighi commented:
@thewchan Hi
Thanks for your efforts 💯 
I made some minor edits in `meta.yaml` (`description` section) and also added two main developers of the `art` library to `recipe-maintainers` section.

SH

thewchan committed:
Update meta.yaml

Thanks for the PR, I have updated the URLs, putting them in the right places in the recipe.

thewchan commented:
@conda-forge-admin please rerender

Commit:
MNT: Re-rendered with conda-build 3.22.0, conda-smithy 3.21.1, and conda-forge-pinning 2022.08.26.18.27.48
